core/savings.go

```diff
@@ -0,0 +1,86 @@
+package core
+
+import (
+	"math"
+	"time"
+
+	"github.com/evcc-io/evcc/util"
+)
+
+// Site is the main configuration container. A site can host multiple loadpoints.
+type Savings struct {
+	log *util.Logger
+
+	startupTime            time.Time // Boot time
+	lastUpdateTime         time.Time // Time of last charged value update
+	chargedTotal           float64   // Energy charged since startup (Wh)
```

Comment:
Alle Zählerwerte sind in kWh. Hier auch konsistent?

Reply:
Werte sind nun überall kWh